docs/quickstart/run_model_on_rtsp_webcam.md

```diff
@@ -67,6 +51,40 @@ Your webcam will open and you can see the model running:
   <source src="https://media.roboflow.com/rock-paper-scissors.mp4" type="video/mp4">
 </video>
 
+!!! tip
+
+    When you run inference on an image, the same augmentations you applied when you generated a version in Roboflow will be applied at inference time. This helps improve model performance.
+
+## Define Custom Prediction Handlers
+
+The `on_prediction` parameter in the `InferencePipeline` constructor allows you to define custom prediction handlers. You can use this to define custom logic for how predictions are processed.
+
+This function provides two parameters:
+
+- `predictions`: A dictionary that contains all predictions returned by the model for the frame, and;
+- `video_frame`: A dataclass that contains:
+    - `image`: The video frame as a NumPy array,
+    - `frame_id`: The frame ID, and;
+    - `frame_timestamp`: The timestamp of the frame.
+
+For example, you can use the following code to print the predictions to the console:
+
+```python
+from inference.core.interfaces.stream.inference_pipeline import InferencePipeline
+
+def on_prediction(predictions, video_frame) -> None:
+    print(predictions)
+    pass
+
+pipeline = InferencePipeline.init(
+    model_id="rock-paper-scissors-sxsw/11",
+    video_reference=0,
+    on_prediction=on_prediction,
+)
+pipeline.start()
+pipeline.join()
+```
+
 ## New stream interface!
```

Comment:
old example should also be left and the information about deprecation should be announced

Reply:
When we deprecate an API, we should be updating our guides to use the new API.